tests/klay_test_blockchain_test.go

```diff
@@ -92,7 +93,7 @@ func NewBCData(maxAccounts, numValidators int) (*BCData, error) {
 	////////////////////////////////////////////////////////////////////////////////
 	// Create a governance
 	gov := generateGovernaceDataForTest()
-
+	gov.SetNodeAddress(nodeAddr)
```

Comment:
Can you explain what this is for?

Reply:
@jimni1222 Sure. When `WriteGovernanceState` is called, it encodes `Governance` instance into a JSON object and as `toJSON` method is as below, if `nodeAddress` is not set, error happens due to type assertion of nil object.

```
func (gov *Governance) toJSON(num uint64) ([]byte, error) {
	ret := &governanceJSON{
		BlockNumber:     num,
		ChainConfig:     gov.ChainConfig,
		VoteMap:         gov.voteMap.Copy(),
		NodeAddress:     gov.nodeAddress.Load().(common.Address), << here
		GovernanceVotes: gov.GovernanceVotes.Copy(),
		GovernanceTally: gov.GovernanceTallies.Copy(),
		CurrentSet:      gov.currentSet.Items(),
		ChangeSet:       gov.changeSet.Items(),
	}
	j, _ := json.Marshal(ret)
	return j, nil
}
```

Reply:
Oh, thank you for explaination :)